src/main/java/Duke.java

```diff
@@ -1,10 +1,272 @@
+import duke.command.Deadline;
+import duke.command.Event;
+import duke.command.Task;
+import duke.command.Todo;
+import duke.exception.*;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
+//    private static Task[] items = new Task[100];
```

Comment:
Don't retain commented code. If necessary, you can recover the code from Git later